src/diffusers/pipelines/stochatic_karras_ve/pipeline_stochastic_karras_ve.py

```diff
@@ -18,9 +16,6 @@ class KarrasVePipeline(DiffusionPipeline):
     differential equations." https://arxiv.org/abs/2011.13456
     """
 
-    unet: UNet2DModel
```

Comment:
@anton-l think we don't want those to be class attributes no? Haven't seen this anywhere else I think

Reply:
Actually I wanted to add them everywhere else instead! They help with type hinting in PyCharm and VSCode, otherwise they get highlighted as missing, since we set class attributes implicitly in `self.register_modules()`

Reply:
Ah yeah I see - good point. Had to look this up actually: https://stackoverflow.com/questions/53897054/python-class-members-type-hinting

Good for me then! Reverting this change